packages/extension/package.json

```diff
@@ -120,7 +120,7 @@
     "tailwindcss": "^3.3.3",
     "terser-webpack-plugin": "^4.2.3",
     "ts-jest": "^26.5.4",
-    "typescript": "^4.6.4",
```

Comment:
Why are we lowering typescript version here?

Reply:
Mainly to uniform it accross our apps, we had 2 different versions, pnpm would resolve the lower one anyway.
But by using this in combination with the docs one it would mess up jest somehow.

Reply:
Can we go higher to `4.6.4` or is it because of jest?

Reply:
I'll try one more time, but had major issues with Jest for extension on the higher version for some reason.

Reply:
@capJavert Jep, upgrading breaks extension jest completly.
I prefer to have this minor bump out then if that's ok with you?

Reply:
Sure 👍